docs/reference/aggregations/bucket/range-field-note.asciidoc

```diff
@@ -0,0 +1,173 @@
+[[search-aggregations-bucket-range-field-note]]
+=== Note on Bucketing Range Fields
+
+==== Documents are counted for each bucket they land in
+
+Since a range represents multiple values, running a bucket aggregation over a
+range field can result in the same document landing in multiple buckets.  This
+can lead to surprising behavior, such as the sum of bucket counts being higher
+than the number of matched documents.  For example, consider this histogram
+over an integer range from 10 to 20: 
+
+[source, console]
+--------------------------------------------------
+PUT range_index
+{
+  "settings": {
+    "number_of_shards": 2
+  },
+  "mappings": {
+    "properties": {
+      "expected_attendees": {
+        "type": "integer_range"
+      },
+      "time_frame": {
+        "type": "date_range",
+        "format": "yyyy-MM-dd HH:mm:ss||yyyy-MM-dd||epoch_millis"
+      }
+    }
+  }
+}
+
+PUT range_index/_doc/1?refresh
+{
+  "expected_attendees" : {
+    "gte" : 10,
+    "lte" : 20
+  },
+  "time_frame" : {
+    "gte" : "2019-10-28 12:00:00",
+    "lte" : "2019-11-04"
+  }
+}
+POST /range_index/_search?size=0
+{
+    "aggs" : {
+        "range_histo" : {
+            "histogram" : {
+                "field" : "expected_attendees",
+                "interval" : 5
+            }
+        }
+    }
+}
+--------------------------------------------------
+
+Since the interval is 5 (and the offset is 0 by default), we expect buckets 10,
+15, and 20. Our range document will fall in all three of these buckets,
+something like this:
+
+[source, console-result]
+--------------------------------------------------
+{
+  ...
+  "aggregations" : {
+    "range_histo" : {
+      "buckets" : [
+        {
+          "key" : 10.0,
+          "doc_count" : 1
+        },
+        {
+          "key" : 15.0,
+          "doc_count" : 1
+        },
+        {
+          "key" : 20.0,
+          "doc_count" : 1
+        }
+      ]
+    }
+  }
+}
+--------------------------------------------------
+
+At present, there is no way for a document to exist partially in a bucket - we
```

Comment:
I might break this up a bit, something like:

> At present, there is no way for a document to exist partially in a bucket - we
cannot count "one-third" in each of the above three buckets. In this example,
since the document's range overlapped all buckets, it will be included in
the sub-aggregations for each bucket as well.

Reply:
👍